NerdDinner/Models/Dinner.cs

```diff
@@ -11,43 +11,38 @@ public partial class Dinner
     {
         
         
-        public bool IsValid
-        {
-            get { return (GetRuleViolations().Count() == 0); }
-        }
+        public bool IsValid => !GetRuleViolations().Any();
```

Comment:
Does the same thing, but a whole lot shorter, for two reasons.
1. Uses expression-bodied member, which is just syntax sugar
2. Uses !.Any instead of count==0.